### DISCUSSION TITLE
Credit Card instalments

### DISCUSSION THREAD

**Discussion by azrossi** [firefly-iii/firefly-iii#5968] [Firefly III]
In Brazil it is very common to split a bill in 2 or more instalments when you pay with credit card.
E.g. You buy a fridge for R$3,000 and you split the payment in 10 instalments of R$300. So your next credit card bill will show -R$300 (1/10), the next month -R$300 (2/10), and so on. This will immediately hold R$3,000 of your credit card limit which will be released accordingly to instalments payments (after you pay the first bill the credit hold will be R$2,700, etc...)  
When you input a credit card cost you decided to split, either you enter 10 (or _N_) costs with respective due dates, or the cashflow will not reflect what's really going on.
Is there an easier way to enter these kind of expenses?

**Comment by JC5** [comment]
Not really, but you could set the "virtual balance" even though you would have to update it manually.

**Reply by MahmutHizal** [reply]
This is one of the most ridiculous responses I've ever seen. It's absurd that such a basic feature requires so much effort to implement. It's also disappointing that the system I installed with high hopes cannot meet such fundamental needs.

**Reply by JC5** [reply]
Sorry, I don't know what I can say? It's all free software, you get what you pay for I guess 🤷‍♀️
